doc/getting-started/apl-nuc.rst

```diff
@@ -33,13 +37,16 @@ below:
    <https://www.intel.com/content/dam/support/us/en/documents/boardsandkits/NUC7i5DN_TechProdSpec.pdf>`__
    for details
 
+Or buy this kind of serial DB9 header cable for NUC:
 
 .. figure:: images/KBL-serial-port-header-to-RS232-cable.jpg
    :align: center
 
    KBL serial port header to RS232 `cable
    <https://www.amazon.com/dp/B07BV1W6N8/ref=cm_sw_r_cp_ep_dp_wYm0BbABD5AK6>`_
 
+And use a RS232 DB9 female to USB cable or RS232 DB9 female/female cable to connect
```

Comment:
About femal/femal cable, there are two kinds (pass through or 2-3 cross). In fact, need 2-3 cross type, can you help to describe it explicitly? thanks! (please refer to https://item.jd.com/12186873066.html) BTW, this picture is right and is fine to me.

Reply:
Add the note for ``crossover`` and ``straight through`` of RS232 DP9 female/female cable.